scripts/ingest.py

```diff
@@ -115,6 +115,10 @@ def parse_args() -> argparse.Namespace:
     parser.add_argument(
         '--no-excise', dest='excise', action='store_false',
         help='disable excision of flagged data [default=no]')
+    parser.add_argument(
+        '--use-data-suspect', dest='use_data_suspect', action='store_false',
```

Comment:
No need to set the destination - the dashes are automatically converted to underscores.

Reply:
Did you mean to use `store_true`? Currently if you specify `--use-data-suspect`, then `use_data_suspect` becomes False.